library/modules/Textures.cpp

```diff
@@ -64,7 +65,7 @@ const uint32_t TILE_WIDTH_PX = 8;
 const uint32_t TILE_HEIGHT_PX = 12;
 
 static size_t load_textures(color_ostream & out, const char * fname,
-                            long *texpos_start) {
+                            long *texpos_start, int tile_w, int tile_h) {
```

Comment:
i am really not fluent in c++... at all. °-°
are these argument/parameter/variable names okay? 

also, is it ok to use `int` as types, here? or should i have used `uint32_t`?

originally i meant to do something like...
```cpp
int tile_w = 8, int tile_h = 12)
```
but i really dont know anything about c++ and didn't know (or wanted to find out) if that actually works. 🤔